ouroboros-consensus/src/ouroboros-consensus/Ouroboros/Consensus/Storage/LedgerDB/Snapshots.hs

```diff
@@ -236,7 +267,7 @@ snapshotToPath = mkFsPath . (:[]) . snapshotToFileName
 snapshotFromPath :: String -> Maybe DiskSnapshot
 snapshotFromPath fileName = do
     number <- readMaybe prefix
-    return $ DiskSnapshot number suffix'
+    return $ DiskSnapshot number suffix' Nothing
```

Comment:
This will be used when listing snapshots, so on startup all of them will have a CRC `Nothing` therefore making this check void. My guess is that we want to have a `<slotno>[_<suffix>].crc` file next to the snapshot and read that one maybe?

Reply:
We also put checksums in a separate file in `lsm-tree`. We use a small abstraction for it:

https://github.com/IntersectMBO/lsm-tree/blob/d11c709cb3c3b91d9f66e94620017e0edd997528/src/Database/LSMTree/Internal/CRC32C.hs#L239-L262